modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,194 @@
+** An overview of the `Repository` CR created on the cluster and the URL associated with the repository.
+** Details of any installed GitHub applications.
++
+With this command, you can also specify a custom GitHub API URL by using the `--github-api-url` argument.
+
+* With this update, error detection is enabled for all `PipelineRuns` by default. {pac} detects if a `PipelineRun` execution has failed and shows a snippet of the last few lines of the error. For a GitHub application, {pac} detects error messages in the container logs and exposes them as annotations on a pull request. 
+
+* This update allows you to fetch tasks from a private {tekton-hub} instance attached to a private Git repository. To enable this update, {pac} uses the internal raw URL of the private {tekton-hub} instance instead of using the GitHub raw URL.
+
+* Before this update, {pac} provided logs that would not include the namespace detail. With this update, {pac} adds the namespace information to the pipeline logs so that you can filter them based on a namespace and debug easily.
+
+* With this update, you can define the provenance source from where the `PipelineRun` definition will be fetched. By default, {pac} fetches the `PipelineRun` definition from the branch where the event has been triggered. Now, you can configure the value of the `pipelinerun_provenance` setting to `default_branch` so that the `PipelineRun` definition will be fetched from the default branch of the repository as configured on GitHub.
+
+* With this update, you can extend the scope of the GitHub token at the following two levels:
+** Repository-level: Use this level to extend the scope to the repositories that exist in the same namespace in which the original repository exists.
+** Global-level: Use this level to extend the scope to the repositories that exist in a different namespace.
+
+* With this update, {pac} triggers a CI pipeline for a pull request created by a user who is not an owner, collaborator, or public member or is not listed in the `owner` file but has permissions to push changes to the repository.
+
+* With this update, the custom console setting allows you to use custom parameters from a `Repository` CR. 
+
+* With this update, {pac} changes all `PipelineRun` labels to `PipelineRun` annotations. You can use a `PipelineRun` annotation to mark a Tekton resource, instead of using a `PipelineRun` label.
+
+
+[id="breaking-changes-1-11_{context}"]	
+== Breaking changes	
+
+* This update replaces the `resource-verification-mode` feature flag with a new flag named `trusted-resources-verification-no-match-policy` in the pipeline specification.
+
+
+[id="deprecated-features-1-11_{context}"]	
+== Deprecated and removed features	
+
+* This update removes support for the `PipelineResource` commands and references from Tekton CLI:
+** Removal of `PipelineResources` from cluster tasks
+** Removal of `PipelineResources` from tasks
+** Removal of `PipelineResources` from pipelines
```

Comment:
mention the object type after the backticks?

Reply:
They are resources only, if i write `PipelineResources` resources, then it would sound odd.
I am making it generic.